zap/src/main/java/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -1289,7 +1289,9 @@ public PluginStats getPluginStats(int pluginId) {
     }
 
     /**
-     * Tells whether or not the message matches the specific {@code CustomPageType}
+     * Tells whether or not the message matches the specific {@code CustomPage.Type}. Falls back to
+     * use {@code Analyser} {@link #isFileExist(HttpMessage)} if the type is {@code
```

Comment:
Should the javadoc here be fore specific about the fallbacks/assumptions since this method uses `Context.isCustomPageWithFallback`?